dockerfiles/init/modules/che-kubernetes-helm/templates/configmap.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: che
+  name: che
+data:
+  CHE_HOST: {{ template "cheHost" . }}
+  CHE_PORT: "8080"
+{{- if .Values.tlsEnabled }}
+  CHE_API: https://{{ template "cheHost" . }}/api
+  CHE_WEBSOCKET_ENDPOINT:  wss://{{ template "cheHost" . }}/api/websocket
+  CHE_INFRA_KUBERNETES_BOOTSTRAPPER_BINARY__URL: https://{{ template "cheHost" . }}/agent-binaries/linux_amd64/bootstrapper/bootstrapper
+{{ else }}
+  CHE_API: http://{{ template "cheHost" . }}/api
+  CHE_WEBSOCKET_ENDPOINT:  ws://{{ template "cheHost" . }}/api/websocket
+  CHE_INFRA_KUBERNETES_BOOTSTRAPPER_BINARY__URL: http://{{ template "cheHost" . }}/agent-binaries/linux_amd64/bootstrapper/bootstrapper
+{{- end }}
+  CHE_DEBUG_SERVER: "true"
+  CHE_INFRASTRUCTURE_ACTIVE: kubernetes
+  CHE_INFRA_KUBERNETES_MACHINE__START__TIMEOUT__MIN: "5"
+  CHE_INFRA_KUBERNETES_MASTER__URL: ""
+  CHE_INFRA_KUBERNETES_OAUTH__TOKEN: ""
+  CHE_INFRA_KUBERNETES_PASSWORD: ""
+  CHE_INFRA_KUBERNETES_USERNAME: ""
+  CHE_INFRA_KUBERNETES_NAMESPACE: ""
+  CHE_INFRA_KUBERNETES_TRUST__CERTS: "false"
+  CHE_INFRA_KUBERNETES_PVC_STRATEGY: "common"
+  CHE_INFRA_KUBERNETES_PVC_PRECREATE__SUBPATHS: "true"
```

Comment:
Please change it to `false` according to https://github.com/eclipse/che/pull/8726